modules/navigation/nav_region.cpp

```diff
@@ -115,11 +115,11 @@ void NavRegion::update_polygons() {
 
 #ifdef DEBUG_ENABLED
 	if (!Math::is_equal_approx(double(map->get_cell_size()), double(mesh->get_cell_size()))) {
-		ERR_PRINT_ONCE("Navigation map synchronization error. Attempted to update a navigation region with a navigation mesh that uses a different `cell_size` than the `cell_size` set on the navigation map.");
+		ERR_PRINT_ONCE(vformat("Navigation map synchronization error. Attempted to update a navigation region with a navigation mesh that uses a `cell_size` of %s while assigned to a navigation map set to a `cell_size` of %s. The cell size for navigation maps can be changed by using the NavigationServer map_set_cell_size() function. The cell size for default navigation maps can also be changed in the ProjectSettings.", double(map->get_cell_size()), double(mesh->get_cell_size())));
```

Comment:
```suggestion
		ERR_PRINT_ONCE(vformat("Navigation map synchronization error. Attempted to update a navigation region with a navigation mesh that uses a `cell_size` of %f while assigned to a navigation map set to a `cell_size` of %f. The cell size for navigation maps can be changed by using the NavigationServer map_set_cell_size() function. The cell size for default navigation maps can also be changed in the ProjectSettings.", double(map->get_cell_size()), double(mesh->get_cell_size())));
```
Needs to be %f for floats here, same below

Reply:
The %f formatter would print 0.250000 instead of simple string conversion with %s that removes all unnecssary decimals and just prints 0.25 (and not e.g.  0.250 when %.3f).

Reply:
Where did you see that behavior? In my experience, `%f` avoids displaying padding zeroes.

Reply:
I first used %f before replacing it with %s when encountering this issue.

Reply:
That's a bug then, as it uses `num` which removes trailing zeroes

Reply:
Could use `String::num(...)` to ensure stability but the shortcomings of `vformat` makes it reasonable to use `%s`, had missed that quirk of the implementation